gen/calc.py

```diff
@@ -1262,6 +1287,12 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'superuser_service_account_uid': '',
         'superuser_service_account_public_key': '',
         '_superuser_service_account_public_key_json': calculate__superuser_service_account_public_key_json,
+        # This is a private variable that is not supposed to be used by cluster
```

Comment:
"This" is easy to misunderstand. Instead of "This" let's write the name of the variable in the code comment. Can be done after merge.